ecf/subseasonal/prep/jevs_subseasonal_prep.ecf

```diff
@@ -0,0 +1,101 @@
+#PBS -N jevs_subseasonal_prep_00
+#PBS -j oe 
+#PBS -o /lfs/h2/emc/vpppg/noscrub/shannon.shields/EVS/ecf/subseasonal/prep/evs_subseasonal_prep_out.txt
+#PBS -S /bin/bash
+#PBS -q "dev"
+#PBS -A VERF-DEV
+#PBS -l walltime=02:00:00
+#PBS -l place=vscatter:exclhost,select=1:ncpus=128:ompthreads=1
+#PBS -l debug=true
+#PBS -V
+
+set -x
+
+export model=evs
+
+cd $PBS_O_WORKDIR
+module reset
+
+export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/EVS
+
+versionfile=$HOMEevs/versions/run.ver
+. $versionfile
+module load prod_envir/2.0.6
+module load prod_util/2.0.13
```

Comment:
use ${prod_envir_ver} and ${prod_util_ver} in place of 2.0.6 and 2.0.13, respectively.